CMakeLists.txt

```diff
@@ -860,7 +860,7 @@ ENDIF()
 
 # Check if supported linear algebra backend is found
 IF (HAVE_CXX0X OR HAVE_CXX11)
-	IF ((EIGEN3_FOUND AND ENABLE_EIGEN) OR (VIENNACL_FOUND AND ENABLE_VIENNACL))
+	IF (HAVE_EIGEN3 OR (VIENNACL_FOUND AND ENABLE_VIENNACL))
```

Comment:
@lambday this should be both ```(HAVE_EIGEN3 OR HAVE_VIENNACL)``` right? since we already check ```( _FOUND and ENABLE_ )``` on top and ```set( HAVE_ )```